docker-entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+# Add elasticsearch as command if needed
+if [[ "$1" == "-"* ]]; then
+  set -- elasticsearch "$@"
+fi
+
+# Drop root privileges if we are running elasticsearch
+if [[ "$1" == "elasticsearch" ]]; then
```

Comment:
Also, indentation in this file looks like it's a two-space indent, but should be tabs. :+1: